src/main/java/com/jcabi/maven/plugin/AjcMojo.java

```diff
@@ -536,12 +543,27 @@ private boolean hasSourceroots() {
      */
     private void copyUnwovenClasses()
         throws MojoFailureException {
-        this.unwovenClassesDir.mkdirs();
-        Logger.info(
-            this, "Unwoven classes will be copied to %s",
-            this.unwovenClassesDir
-        );
         if (this.hasClasses()) {
```

Comment:
@amihaiemil I think this method get too complicated now, we need to split somehow or even extract some class. I'm OK with a puzzle